public/angular/templates/incidents/index.html

```diff
@@ -56,7 +56,7 @@
     <button class="btn btn-default" aggie-confirm="Are you sure you want to delete selected incidents?" on-confirm="removeSelected()">Delete</a>
   </div>
   <div class="pull-left margin-right padding">
-    <button ng-controller="IncidentFormModalController" class="btn btn-primary" ng-click="create()">Create Incident</a>
+    <button ng-controller="IncidentFormModalController" class="btn btn-info" ng-click="create()">Create new incident</button>
```

Comment:
Why did this change? The text should be Title Case, and it now runs off the end of the button. The old wording was better I think.